src/apis/summary.js

```diff
@@ -1,5 +1,6 @@
 import axios from 'axios';
 
 export default axios.create({
-  baseURL: 'https://summary-api.datamermaid.org/v1'
+  // baseURL: 'https://summary-api.datamermaid.org/v1'
+  baseURL: 'http://localhost:8081/v1/'
```

Comment:
Server backend locations are generally better-handled from an .env file. This is a common practice in most application development. 

Consider making an .env file in the root directory. Then enter the following:

```sh
REACT_APP_API_URL="http://localhost:8080" 
```

Then replacing this code with the following:

```js
  baseURL: `${process.env.REACT_APP_API_URL}/v1/`
```

Also worth adding a note about this variable in the readme as well :)